dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -178,7 +178,7 @@ export default defineComponent({
           ? kpiValue.map((naceCodeShort: string) => naceCodeMap.get(naceCodeShort)?.label ?? naceCodeShort)
           : naceCodeMap.get(kpiValue as string)?.label ?? kpiValue;
```

Comment:
In line 106 I'd appreciate it if we name the variable something like this:

`kpiValueFormated = this.reformatValueForDisplay(kpiField, kpiValue);`

And use the `kpiValueFormated` in the following lines inside of the `createKpiDataObjects()` method.

That way it is clearer to the reader that the fromated value is used, and not the "raw" one.

Reply:
Addition:

We could even do:
`content: { [dataIdOfLksgDataset]: this.reformatValueForDisplay(kpiField, kpiValue);},`

in line 115 since the `kpiValue` is only used there.

Reply:
👍